cmd/samples/httpb/receiver-direct/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/http"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+)
+
+func main() {
+	ctx := context.Background()
+	p, err := cloudevents.NewHTTP()
+	if err != nil {
+		log.Fatalf("failed to create protocol: %s", err.Error())
+	}
+
+	h, err := cloudevents.NewHTTPReceiveHandler(ctx, p, receive)
```

Comment:
That's coool!

Reply:
I agree. I can see it working with the queues too